### PR TITLE
kube-burner-ocp integration

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description

Integrating `kube-burner-ocp` v1.0.0, built using the most recent code.

This "new" binary uses the code from the https://github.com/kube-burner/kube-burner-ocp repository, repository depending on the original `kube-burner` repo.

An ADR doc is pending from my side to be written to track this change. 

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: workloads/kube-burner-ocp-wrapper/run.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ LOG_LEVEL=${LOG_LEVEL:-info}
 if [ "$KUBE_BURNER_VERSION" = "default" ]; then
     unset KUBE_BURNER_VERSION
 fi
-KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.8.1}
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.0.0}
```
Comment (venkataanil) [inline workloads/kube-burner-ocp-wrapper/run.sh:10]: Let's rename it to KUBE_BURNER_OCP_VERSION to reflect kube-burner-ocp release

**Review by venkataanil — APPROVED** [review]
LGTM

**Review by mohit-sheth — APPROVED** [review]
lgtm

**Review by krishvoor — APPROVED** [review]
/lgtm

**Review by jtaleric — CHANGES_REQUESTED** [review]
I would like to see us verify this doesn't break *anything* in prow.

**Comment by jtaleric** [comment]
things I think it will break are the jobs which use legacy workloads that QE depends on.
 
 cc @paigerube14 @nathan-weinberg @svetsa-rh

**Comment by svetsa-rh** [comment]
> I would like to see us verify this doesn't break anything in prow.
@paigerube14 Anything in Jenkins as well that will break because of this change?
@jtaleric AFAIK, our tests in Prow use ocp version anyway. Also, if we want to verify nothing in Prow is breaking because of this change, we set KUBE_BURNER_VERSION explicitly and try? Or something more?

**Comment by paigerube14** [comment]
@svetsa-rh no, ran a quick test and no changes are needed in jenkins since we're updating the run script that we call 
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/kube-burner-ocp/899/console

**Comment by nathan-weinberg** [comment]
if the PerfScale QE job is still working this should be fine for NetObserv QE as well - thanks for checking @jtaleric!

**Comment by rsevilla87** [comment]
> I would like to see us verify this doesn't break _anything_ in prow.

Let's try it using rehearsals

**Review by jtaleric — APPROVED** [review]
lgtm

**Pull request merged**